gadget/mkfs.go

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-package internal
+package gadget
```

Comment:
then the package would be `mkfs` and the functions exported would be named as follows:

* `internal.MkfsFunc` -> `mkfs.MakeFunc`
* `internal.Mkfs` -> `mkfs.Make`
* `internal.MkfsWithContent` -> `mkfs.MakeWithContent`

Reply:
Done! Thanks for the feedback.